10up/headstartwp
# App Router Preview Handler

### Description of the Change



Partially addresses #791 

### How to test the Change
- `npm install`
- `npm run dev:app`
- Make sure your wp-env has its Headless URL pointing to `http://localhost:3000`
- Visit `localhot:8888/wp-admin` and preview both draft and published posts.

https://github.com/10up/headstartwp/assets/6104632/de26affc-6ebd-454c-879e-754124af9a8a



### Checklist:
